spark/components/masthead/vanilla/masthead.js

```diff
@@ -1,7 +1,10 @@
 /* global document window */
+import { throttle } from 'lodash';
 import getElements from '../../../utilities/helpers/vanilla/getElements';
 import { focusFirstEl } from '../../../utilities/helpers/vanilla/elementState';
 import { isEscPressed } from '../../../utilities/helpers/vanilla/keypress';
+import isElementVisible from '../../../utilities/helpers/vanilla/isElementVisible'
```

Comment:
```suggestion
import isElementVisible from '../../../utilities/helpers/vanilla/isElementVisible;'
```
Missing some semi-colons here.